packages/ilib-loctool-android-layout/package.json

```diff
@@ -40,12 +40,12 @@
     ],
     "repository": {
         "type": "git",
-        "url": "https://github.com/iLib-js/ilib-loctool-android-layout.git"
+        "url": "https://github.com/iLib-js/ilib-mono/ilib-loctool-android-layout.git"
     },
     "scripts": {
-        "dist": "npm pack",
-        "test": "jest",
-        "test:watch": "jest --watch",
+        "dist": "pnpm pack",
+        "test": "pnpm jest",
+        "test:watch": "pnpm jest --watch",
         "debug": "node --inspect-brk node_modules/.bin/jest -i",
```

Comment:
`debug` should be updated to
```sh
NODE_OPTIONS=\"$NODE_OPTIONS --inspect-brk\" jest -i
```
similar to https://github.com/iLib-js/ilib-mono/blob/main/packages/ilib-lint/package.json#L55